en/uavcan/README.md

```diff
@@ -71,6 +75,7 @@ In order to use UAVCAN components with PX4 you will first need to enable the UAV
 1. [UAVCAN_ENABLE](../advanced_config/parameter_reference.md#UAVCAN_ENABLE) must be [set](../advanced_config/parameters.md) to one of the non-zero values.
```

Comment:
@coderkalyan In particular this info is the PX4 configuration. It gets referenced everywhere and must be visible after the update.